streams/src/main/java/org/apache/kafka/streams/kstream/internals/graph/TableSourceNode.java

```diff
@@ -31,7 +35,7 @@
  */
 public class TableSourceNode<K, V, S extends StateStore> extends StreamSourceNode<K, V> {
 
-    private final StoreBuilder<S> storeBuilder;
+    private final MaterializedInternal<K, V, ?> materializedInternal;
```

Comment:
Pass in Materialized instead of StoreBuilder since we may only `logically` materialize now.

Reply:
The `?` is for covering the known to-do that only k-v store is allowed for source KTable today (see below).